cvmfs/mountpoint.cc

```diff
@@ -1783,6 +1792,19 @@ bool MountPoint::SetupBehavior() {
   LogCvmfs(kLogCvmfs, kLogDebug, "kernel caches expire after %d seconds",
            static_cast<int>(kcache_timeout_sec_));
 
+  uint64_t statfs_time_cache_valid = 0;
+  if (options_mgr_->GetValue("CVMFS_STATFS_CACHE_TIMEOUT", &optarg)) {
+    // 0ul does not work because of ubuntu 18
+    if ( optarg.size() > 0 ) {
```

Comment:
That's a good point! But I'd rather like handle this in `String2Uint64`, which should return 0 for the empty input string. This would leave us in a well-defined situation for the other numeric parameters, too.

Reply:
have a look in the new commit. if this kinda fits your idea this could also be implemented for `String2Int64` and `String2Uint64Pair` 

Short info about the return value of `sscanf`:
*On success, the function returns the number of items in the argument list successfully filled. This count can match the expected number of items or be less (even zero) in the case of a matching failure.
In the case of an input failure before any data could be successfully interpreted, [EOF](https://cplusplus.com/EOF) is returned.*